lib/plugins/state-flow.js

```diff
@@ -7,7 +7,9 @@ function register(skill) {
   });
 
   skill.onAfterStateChanged((alexaEvent, reply, transition) => {
-    alexaEvent.flow.push(transition.to);
+    if (transition) { // unhandled state
```

Comment:
So I get that we're null-checking this now, but I'm not sure if onAfterStateChanged should even be called if there is no transition. Isn't this what onUnhandled is for? If we do choose to make transition sometimes null, we need to make that loud and clear in the docs, and explain what circumstances make it null.